ee/ucp/admin/configure/deploy-route-reflectors.md

```diff
@@ -1,137 +1,133 @@
 ---
-title: Improve network performance with Route Reflectors
-description: Learn how to deploy Calico Route Reflectors to improve performance
-  of Kubernetes networking
+title: Improve network performance with route reflectors
+description: Learn how to deploy Calico route reflectors to improve the performance of Kubernetes networking.
```

Comment:
A suggestion: "the performance of Kubernetes networking." --> "Kubernetes networking performance."